### ISSUE TITLE
Division of LIDC-IDRI dataset?

### ISSUE THREAD

**Issue by tricoffee** [ymli39/DeepSEED-3D-ConvNets-for-Pulmonary-Nodule-Detection#5]
How to use LIDC-IDRI DataSet? How to divide LIDC-IDRI  into training set and test set? What does alllabelfiles mean? Where is this file "/home/LungNodule_DL/LIDC/labels/new_nodule.csv" from?

**Comment by ymli39** [comment]
You could randomly split the LIDC dataset, depends on your preference. alllabelfiles is the file that contains all gt bounding boxes for LIDC dataset. new_nodule.csv is the file I generated by mining xml file provided by LIDC. Please allow me to modify your question, I should not put personally name there.
